File: go/1074.go
package main

import (
	"fmt"
	"math"
)

func main() {
  var n, v int
  fmt.Scanf("%d", &n)

  for i := 0; i < n; i++ {
    fmt.Scanf("%d", &v)

    if v == 0 {
      fmt.Printf("NULL\n")
      continue
    }
    
    if v % 2 == 0 {
      fmt.Printf("EVEN ")
    }

    if math.Abs(float64(v % 2)) == 1 {
      fmt.Printf("ODD ")
    }

    if v > 0 {
      fmt.Printf("POSITIVE\n")
    }

    if v < 0 {
      fmt.Printf("NEGATIVE\n")
    }
  }
}

File: lua/1074.lua
local n = io.read("*n")

for i = 1, n do
    local v = io.read("*n")

    if v == 0 then
        print("NULL")
    elseif v % 2 == 0 then
        io.write("EVEN ")
    elseif math.abs(v % 2) == 1 then
        io.write("ODD ")
    end

    if v > 0 then
        print("POSITIVE")
    elseif v < 0 then
        print("NEGATIVE")
    end
end
